packages/react-native-editor/__device-tests__/helpers/caps.js

```diff
@@ -4,7 +4,7 @@ const ios = {
 	os: 'iOS',
 	deviceOrientation: 'portrait',
 	automationName: 'XCUITest',
-	appiumVersion: '1.20.1', // Sauce Labs requires appiumVersion to be specified.
+	appiumVersion: '1.22.0', // Sauce Labs requires appiumVersion to be specified.
```

Comment:
I noticed that on GB-mobile we didn't update Appium to `1.22.0`, so this change might affect the E2E tests in that repo. For this reason, I've opened [a PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4284) to update it.

Reply:
Looks like we need to make further changes in the GB-mobile E2E tests to use this Appium version, as I triggered a run and failed with the following error ([reference](https://app.circleci.com/pipelines/github/wordpress-mobile/gutenberg-mobile/15113/workflows/1a972cbf-dbe3-482e-b5dd-31c257034000/jobs/82161?invite=true#step-118-21)):

```
Appium 1.22.0 does not support iOS 14.5. Please check our platform configurator (https://saucelabs.com/docs/platforms)
```

I don't want to block this PR due to this change, so I'm going to revert it and address it in a separate PR.

Reply:
Change reverted in [this commit](https://github.com/WordPress/gutenberg/pull/35879/commits/1faa89b0e2b1b31dcf357a861793e716b911d960).